app/js/shortcuts.js

```diff
@@ -0,0 +1,188 @@
+module.exports = {};
+
+(function () {
+  "use strict";
+  var curShortcuts = {},
+      allShortcuts = {},
+      activeGroups = [],
+      pausedGroups = [],
+
+      // All of the features that can be set as keyboard shortcuts.
+      features = {
+        // Features in the main window.
+        main: {
+          takePicture: function() {
+            btnCaptureFrame.click();
```

Comment:
Highly surprised variables like `btnCaptureFrame` and `playAudio` are available in this scope, TBH.